integration/containerd/cri/integration-tests.sh

```diff
@@ -313,13 +313,21 @@ TestContainerMemoryUpdate() {
 	testContainerStop
 }
 
+# k8s may restart docker which will impact on containerd stop
+stop_containerd() {
+	tmp=$(pgrep kubelet)
+	[ -n "$tmp" ] && sudo kubeadm reset -f
+
+	sudo systemctl stop containerd
+}
+
 main() {
 
 	info "Stop crio service"
 	systemctl is-active --quiet crio && sudo systemctl stop crio
 
 	info "Stop containerd service"
-	systemctl is-active --quiet containerd && sudo systemctl stop containerd
+	systemctl is-active --quiet containerd && stop_containerd
```

Comment:
Why not:

```
systemctl is-active --quiet containerd && ( sudo systemctl stop containerd || true)
```

Grepping for `kubelet` is looking for a special side effect of `containerd`, but there may be other things running (e.g. `etcd`).

Reply:
thanks @c3d -, Yeah, you are right, there may be any process that may change the status of containerd. So there is no way to stop this theoretically. However we can use enumeration to avoid the failure in most cases.
I'm not sure what will happen if containerd doesn't stop. If there is nothing worse, this change is good for me.